test/manual/amp-analytics/amp-analytics-multi-selector.html

```diff
@@ -72,6 +77,15 @@
         data-ad-height=250>
     </amp-ad>
 
+    <amp-ad id='ad3' width=300 height=250
+        type="_ping_"
+        data-url='https://lh3.googleusercontent.com/pSECrJ82R7-AqeBCOEPGPM9iG9OEIQ_QXcbubWIOdkY=w400-h300-no-n'
+        data-valid='true'
+        data-vars-id='ad3'
+        data-ad-width=300
+        data-ad-height=250
+        class="myClass">
```

Comment:
nit: You can add `.myClass` to another component to make sure multiple elements got selected.